python/VERSION

```diff
@@ -1 +1 @@
-0.11.1
+0.12.0rc0
```

Comment:
I removed the dash since this needs to comply with the Python package versioning requirement: https://packaging.python.org/en/latest/specifications/version-specifiers/#public-version-identifiers